resources/public/api/conf/1.0/subscribe/application.yaml

```diff
@@ -1435,7 +1430,7 @@ components:
             $ref: '#/components/schemas/ValidateProductAuthorisationRequest'
     EntityGroup:
       type: string
-      enum: [ 'UK Record', 'EU Trader', 'EU Warehouse', 'EU Temporary Authorisation' ]
+      enum: [ 'UK Record', 'EU Trader', 'EU Warehouse', 'UK Temporary Authorisation', 'EU Temporary Authorisation' ]
```

Comment:
Maybe highlight with helen? Not a breaking change, but just in case someone goes "UK Temporary Authorisations never existed before!!!"